src/App.jsx

```diff
@@ -2,43 +2,26 @@ import React from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 import './App.scss';
+import { useState } from 'react';
+import { Tabs } from './components/Tabs'
+import { activeTabFinder } from './middlewares/activeTabFinder'
 
 export const tabs = [
   { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
   { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
   { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
 ];
 
-export const App = () => (
-  <div className="section">
-    <h1 className="title">Selected tab is Tab 1</h1>
+export const App = () => {
+  const [activeTabId, setActiveTabId] = useState(tabs[0].id)
 
-    <div data-cy="TabsComponent">
-      <div className="tabs is-boxed">
-        <ul>
-          <li className="is-active" data-cy="Tab">
-            <a href="#tab-1" data-cy="TabLink">
-              Tab 1
-            </a>
-          </li>
+  return (
+    <div className="section">
+      <h1 className="title">
+        Selected tab is {activeTabFinder(tabs, activeTabId)?.title}
```

Comment:
bad practice use func in html
let's move it before return
```suggestion
        Selected tab is {activeTab.title}
```

Reply:
+ you also return some tab so `?`  is unnecessary